tests/fitting/test_fitting_flows.py

```diff
@@ -361,9 +361,9 @@ def test_mlip_fit_maker_glue_xml(
     # Test to check if gap fit runs with default hyperparameter sets (i.e. include_two_body and include_soap is True)
     gapfit = MLIPFitMaker(
         mlip_type="GAP",
+        glue_file_path=glue_file
     ).make(
         species_list=["Si"],
-        isolated_atoms_energy=[-0.82067307],
```

Comment:
Why removed?

Reply:
First reason is that the data type changed to dict and second is that for GAP we retrieve the energy from the isolated atom energy calculation. This keyword is needed for the other MLIP architectures.